package.json

```diff
@@ -44,20 +44,26 @@
     "zod": "^3.23.8"
   },
   "devDependencies": {
+    "@celo/devchain-anvil": "12.0.0-canary.49",
     "@tanstack/eslint-plugin-query": "^5.28.6",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^16.1.0",
     "@types/dompurify": "^3",
     "@types/node": "^20.11.30",
     "@types/react": "^18.2.73",
     "@types/react-dom": "^18.2.22",
     "@typescript-eslint/eslint-plugin": "^7.4.0",
     "@typescript-eslint/parser": "^7.4.0",
+    "@viem/anvil": "^0.0.10",
```

Comment:
would you mind trying suggested [library](https://github.com/wevm/prool)? as per [the github page](https://github.com/wevm/anvil.js) this one is deprecated

of course only if this is not much of a hassle - we could reuse it in other repos, if it is then don't bother

Reply:
Will give it a shot

Reply:
yeah if its a hastle we can merge now and then replace next year

Reply:
yeah I'm running into issues as prool only supports node 22 and I don't think this is the move to try to upgrade right now. (one example issue: https://github.com/wevm/prool/issues/16)